include/proxy-wasm/wasm.h

```diff
@@ -137,6 +139,10 @@ class WasmBase : public std::enable_shared_from_this<WasmBase> {
 
   const std::unordered_map<std::string, std::string> &envs() { return envs_; }
 
+  WASIFileSystem &fs() {
```

Comment:
q: can this be const ref? 
```suggestion
  const WASIFileSystem &fs() {
```

Reply:
Don't think so since we mutate to keep track of opened files